lib/cocoapods/generator/copy_resources_script.rb

```diff
@@ -126,6 +128,7 @@ def script
       xcrun mapc "${PODS_ROOT}/$1" "${CONFIGURATION_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/`basename "$1" .xcmappingmodel`.cdm"
       ;;
     *.xcassets)
+      XCASSET_FILES="$XCASSET_FILES '$1'"
```

Comment:
Should be enough to do it once, if needed at all. Do you see any scenario, when there could be duplicates and know about if that's not already handled by `actool`?